config/srcSchemaMap.py

```diff
@@ -0,0 +1,209 @@
+config.srcSchemaMap = {
+"deblend_nChild": "deblend_nchild",
```

Comment:
The dots get persisted as underscores in the output schema, so it's really just the name churn that needs accommodating.